datasafe-encryption/datasafe-encryption-impl/src/main/java/de/adorsys/datasafe/encrypiton/impl/cmsencryption/CMSEncryptionServiceImpl.java

```diff
@@ -51,6 +58,25 @@ public OutputStream buildEncryptionOutputStream(OutputStream dataContentStream,
         return streamEncrypt(dataContentStream, rec, encryptionConfig.getAlgorithm());
     }
 
+    @Override
+    @SneakyThrows
+    public OutputStream buildEncryptionOutputStream(OutputStream dataContentStream, List<PublicKeyIDWithX509Cert> publicKeyIDWithCertMap) {
```

Comment:
Function buildEncryptionOutputStream on line 53 should simly accept list of PublicKey

Reply:
done